### PR TITLE
Add/pos docs

### PR DESCRIPTION
**Pull request by kristoferlund**
This PR adds documentation for the ckBTC Point of Sale app. The original repo can be found here: https://github.com/kristoferlund/ic-pos

A PR has been opened for inclusion in the Motoko examples library: https://github.com/dfinity/examples/pull/614

> IC-POS is an experimental app to demonstrate a real world use case for ckBTC on the Internet Computer. It is a simple Point of Sale app that allows users to accept ckBTC payments.

### CODE REVIEW

File: docs/samples/pos.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        trigger_threshold = 2000;
+        num_blocks_to_archive = 1000;
+        controller_id = principal "'${OWNER}'";
+      }
+    }
+  })
+'
+```
+
+### Step 4: Save the ledger principal as a variable
```
Comment (jessiemongeon1) [inline docs/samples/pos.md:105]:
```suggestion
### Step 5: Save the ledger principal as a variable.
```

File: docs/samples/pos.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+We need this information in the next step, when deploying the index canister.
+
+```bash
+export LEDGER_PRINCIPAL=$(dfx canister --network local id icrc1-ledger)
+```
+
+### Step 5: Deploy the index canister
```
Comment (jessiemongeon1) [inline docs/samples/pos.md:113]:
```suggestion
### Step 6: Deploy the index canister.
```

File: docs/samples/pos.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+```bash
+dfx deploy --network local icrc1-index --argument '
+  record {
+   ledger_id = (principal "'${LEDGER_PRINCIPAL}'");
+  }
+'
+```
+
+### Step 6: Deploy the icpos canister
```
Comment (jessiemongeon1) [inline docs/samples/pos.md:125]:
```suggestion
### Step 7: Deploy the icpos canister.
```

File: docs/samples/pos.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+The icpos canister manages the store configuration and sends notifications when a payment is received.
+
+The `--argument '(0)'` argument is used to initialize the canister with `startBlock` set to 0. This is used to tell the canister to start monitoring the ledger from block 0. When deploying to the IC mainnet, this should be set to the current block height to prevent the canister from processing old transactions.
+
+```bash
+dfx deploy --network local icpos --argument '(0)'
+```
+
+### Step 7: Configure the icpos canister
```
Comment (jessiemongeon1) [inline docs/samples/pos.md:135]:
```suggestion
### Step 8: Configure the icpos canister.
```

File: docs/samples/pos.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+dfx canister --network local call icpos setLedgerId "${LEDGER_PRINCIPAL}"
+```
+
+ic-pos uses [Courier](https://courier.com/) to send email and SMS notifications. If you want to enable notifications, you need to sign up for a Courier account and and create and API key. Then issue the following command:
+
+```bash
+dfx canister --network local call icpos setCourierApiKey "pk_prod_..."
+```
+
+### Step 8 - Setup `.env` for the frontend environment
```
Comment (jessiemongeon1) [inline docs/samples/pos.md:149]:
```suggestion
### Step 9: Setup `.env` for the frontend environment.
```

File: docs/samples/pos.md [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+
+The frontend needs information about how to access the canisters. This is done by setting environment variables. Copy the `.env.template` file and rename it to `.env`. Then update the values to match your local setup.
+
+```bash
+cp src/.env.template src/.env
+```
+
+### Step 9: Build and run the frontend
```
Comment (jessiemongeon1) [inline docs/samples/pos.md:157]:
```suggestion
### Step 10: Build and run the frontend.
```

File: docs/samples/pos.md [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+Run yarn to install dependencies and start the frontend. The frontend will be available at http://localhost:5173.
+
+```bash
+yarn
+yarn dev
+```
+
+Why don't we deploy the frontend as a local canister? Vite uses lazy loading of modules. This does not work when deploying to a local canister. When deploying to the IC mainnet, this is not an issue. Also, running using `yarn dev` allows for hot reloading of the frontend code when making changes.
+
+### Step 10: Make a transfer!
```
Comment (jessiemongeon1) [inline docs/samples/pos.md:168]:
```suggestion
### Step 11: Make a transfer.
```

**Comment by kristoferlund** [comment]
@jessiemongeon1 Thanks for taking a look at the text! I believe I resolved all corrections. Please note that the app this text is referring to have not yet been added to the examples repo. A PR is pending.

**Comment by kristoferlund** [comment]
@jessiemongeon1 Now PR have been merged in the examples repo and docs page updated to reflect latest changes.

FYI @letmejustputthishere

**Pull request merged**